docs/src/components/pricing/HeroPricing.tsx

```diff
@@ -0,0 +1,34 @@
+import * as React from 'react';
+import Box from '@material-ui/core/Box';
+import Container from '@material-ui/core/Container';
+import Divider from '@material-ui/core/Divider';
+import Typography from '@material-ui/core/Typography';
+import GradientText from '../typography/GradientText';
```

Comment:
absolute path
```suggestion
import GradientText from 'docs/src/components/typography/GradientText';
```

x the others